prac_05/word_occurrences.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404 - Prac 5
+Program: Word Occurrences
+Estimate: 20 minutes
+Actual:   25 minutes
+"""
+
+from operator import itemgetter
```

Comment:
Since itemgetter is only used once (for sorting), this import might not be strictly necessary. You could replace itemgetter with lambda x: x[1], which achieves the same result without an import.